x-eithert/src/X/Control/Monad/Trans/Either.hs

```diff
@@ -11,6 +11,7 @@ module X.Control.Monad.Trans.Either (
   , mapEitherT
   , hoistEither
   , eitherT
+  , tryEitherT
```

Comment:
This shouldn't be here, as it is an extension. (i.e. not available in `Control.Monad.Trans.Either`)